### PR TITLE
#390 Reset a person's score when it falls below 200

### PR DESCRIPTION
**Pull request by carlosmiranda**
#390 

* Reset score when it falls below 200.
* Added puzzle to remove poor performer from people.xml.

### CODE REVIEW

**Comment by carlosmiranda** [comment]
@fabriciofx would appreciate your attention here

File: src/main/resources/com/zerocracy/stk/pm/staff/awards/reject_people_below_score.groovy [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import com.zerocracy.pm.ClaimOut
+import com.zerocracy.pmo.Awards
+
+def exec(Project project, XML xml) {
+  new Assume(project, xml).notPmo()
+  new Assume(project, xml).type('Award points were added')
+  ClaimIn claim = new ClaimIn(xml)
+  String login = claim.param('login')
+  Awards awards = new Awards(project, login).bootstrap()
+  if (awards.total() <= -200) {
```
Comment (fabriciofx) [inline src/main/resources/com/zerocracy/stk/pm/staff/awards/reject_people_below_score.groovy]: @carlosmiranda `awards.total()` is used more than 3 times. WDYT store it in a variable and use it?

File: src/main/resources/com/zerocracy/stk/pm/staff/awards/reject_people_below_score.groovy [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    // @todo 390:30min We should remove people from people.xml
+    //  when their score goes below 200. Let's implement that.
+    //  We should also notify the person in case that happens.
+    String job = claim.param('job')
+    String reason = new Par('@%s: Score of %d is too low and will be reset.')
+      .say(login, awards.total())
```
Comment (fabriciofx) [inline src/main/resources/com/zerocracy/stk/pm/staff/awards/reject_people_below_score.groovy]: @carlosmiranda As explained above.

File: src/main/resources/com/zerocracy/stk/pm/staff/awards/reject_people_below_score.groovy [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    Integer points = -awards.total()
```
Comment (fabriciofx) [inline src/main/resources/com/zerocracy/stk/pm/staff/awards/reject_people_below_score.groovy]: @carlosmiranda As explained above.

**Comment by carlosmiranda** [comment]
@fabriciofx pushed new commits, please check

**Comment by fabriciofx** [comment]
@yegor256 This PR seems fine to me. I accept it.

**Comment by yegor256** [comment]
@rultor merge

**Comment by carlosmiranda** [comment]
@yegor256 can we try again? Fixed XML formatting

**Comment by yegor256** [comment]
@rultor merge

**Comment by carlosmiranda** [comment]
@yegor256 can we try again? Merged from master

**Comment by yegor256** [comment]
@rultor merge

**Comment by carlosmiranda** [comment]
@yegor256 let's retry... I fixed puzzle formatting in the latest commit.

**Comment by 0crat** [comment]
The user @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) resigned from [#442](https://github.com/zerocracy/farm/issues/442), please stop working

**Comment by 0crat** [comment]
This pull request [#442](https://github.com/zerocracy/farm/issues/442) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by yegor256** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/442#issuecomment-361556908))

@yegor256 This is what I know about this job, as in [§32](http://datum.zerocracy.com/pages/policy.html#32):

  * The job [#442](https://github.com/zerocracy/farm/issues/442) is [in scope](http://datum.zerocracy.com/pages/policy.html#14) for 8days
  * The role is `REV`
  * The job is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) for [7hr](http://datum.zerocracy.com/pages/policy.html#8)
  * There is a monetary reward attached
  * @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) will get 0.20 of their hourly rate for this job
  * Some equity will be [vested](http://datum.zerocracy.com/pages/policy.html#37) on completion
  * The job doesn't have any [impediments](http://datum.zerocracy.com/pages/policy.html#9)
  * The [budget](http://datum.zerocracy.com/pages/policy.html#4) is 15 minutes
  * These users are banned and won't be assigned:
    * @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda): This user reported the ticket
    * @fabriciofx[/z](https://www.0crat.com/u/fabriciofx): User was resigned from the ticket

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was successfully finished: +15 points just awarded to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), total is [+420](https://www.0crat.com/u/fabriciofx)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://datum.zerocracy.com/pages/policy.html#28): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+8960](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#442](https://github.com/zerocracy/farm/issues/442) is now out of scope

**Pull request merged**